base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -1195,7 +1195,7 @@ FormatByteSize(LONGLONG cbSize, LPWSTR pwszResult, UINT cchResultMax)
 LPWSTR
 FormatFileSizeWithBytes(const PULARGE_INTEGER lpQwSize, LPWSTR pwszResult, UINT cchResultMax)
 {
-    UINT cchWritten;
+    SIZE_T cchWritten;
```

Comment:
Note: FormatByteSize() function returns an UINT, do you think it also needs to be changed?

Reply:
I changed it to using an extra size_t variable. It's not perfect, but blame the shitty MS interfaces sometimes using ULONG, sometimes SIZE_T (and sometimes size_t which isn't even the same...)